tests/base/time_stepping_01.cc

```diff
@@ -266,6 +266,32 @@ int
 main()
 {
   initlog();
+  deallog.precision(4);
+  {
+    const std::vector<double> bi = {{1153189308089. / 22510343858157.,
+                                     1772645290293. / 4653164025191.,
+                                     -1672844663538. / 4480602732383.,
+                                     2114624349019. / 3568978502595.,
+                                     5198255086312. / 14908931495163.}};
+    std::vector<double>       ai;
+    ai = {{970286171893. / 4311952581923.,
+           6584761158862. / 12103376702013.,
+           2251764453980. / 15575788980749.,
+           26877169314380. / 34165994151039.}};
+
+    deallog << "Check low-storage Runge-Kutta coefficients" << std::endl;
```

Comment:
maybe put this into a separate time_stepping_02.cc test.

Reply:
time_stepping_02.cc is already there, so I will create  time_stepping_03.cc for this. 
To summarize, we will use `Assert(std::fabs(a[i]-bi[i])<1e-10, ExcInternalError());` and only print the difference of c,
correct?

Reply:
that makes sense to me, yes.